src/libcgroup.map

```diff
@@ -146,4 +146,5 @@ CGROUP_3.0 {
 	cgroup_cgxset;
 	cgroup_version;
 	cgroup_list_mount_points;
+	cgroup_setup_mode;
```

Comment:
Nitpick - we should probably add a comment above this method.  (The map file doesn't support naming like `CGROUP_3.0.1`.  Not sure why.)

```
cgroup_list_mount_points;

/* libcgroup 3.0.1 */
cgroup_setup_mode;
```

Reply:
@drakenclimber Thanks for the review. Sure, will include it in the next version of the patch. It is interesting why the naming is supported.